app/models/transaction.rb

```diff
@@ -1,4 +1,9 @@
 class Transaction < ApplicationRecord
   belongs_to :subscription
   belongs_to :user
+
+  def self.daily_transaction
+    subs = Subscription.all.where(billing_day: Date.today.day)
+    subs.each(&:charge_fee)
+  end
```

Comment:
it is not advised to use class methods like this.
Build a rake task for this function.

Reply:
also, this can be a one liner.